src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -338,6 +346,43 @@ public function callableMethodCommand(InputInterface $input, OutputInterface $ou
     {
         $output->writeln('from the code...');
     }
+
+    public function testSetProcessTitle()
+    {
+        $command = new TestCommand();
+
+        $this->assertSame('', cli_get_process_title());
+
+        $command->setProcessTitle('process title');
+        $inputMock = $this->prophesize(InputInterface::class);
+        $outputMock = $this->prophesize(OutputInterface::class);
+        $command->run($inputMock->reveal(), $outputMock->reveal());
+
+        $this->assertSame('process title', cli_get_process_title());
+    }
+
+    public function testSetAliases()
+    {
+        $command = new TestCommand();
+        $command->setAliases([]);
+        $command->setAliases(new TraversableDummy());
```

Comment:
If success, it tests that array and traversable are allowed arguments.

How do you recommend to change it, so it's both tested and readable?